storage/BasicColumnStoreTupleStorageSubBlock.cpp

```diff
@@ -173,45 +173,45 @@ BasicColumnStoreTupleStorageSubBlock::BasicColumnStoreTupleStorageSubBlock(
   }
 }
 
-bool BasicColumnStoreTupleStorageSubBlock::DescriptionIsValid(
+int BasicColumnStoreTupleStorageSubBlock::DescriptionIsValid(
     const CatalogRelationSchema &relation,
     const TupleStorageSubBlockDescription &description) {
   // Make sure description is initialized and specifies BasicColumnStore.
   if (!description.IsInitialized()) {
-    return false;
+    return -1;
```

Comment:
No. If a proto is invalid, we just stop the program and print the proto for debug purpose.